KarhooUISDK/Extensions/UIColor+Extensions.swift

```diff
@@ -85,4 +86,14 @@ extension UIColor {
             return UIColor(cgColor: properColor)
         */
     }
+    
+    func getColor() -> Color {
```

Comment:
SwiftUIs Color has `init(_ color: UIColor)` constructor. Isn't it available in iOS 13?

Reply:
Unfortunately it's been introduced with iOS 15, so it won't be available to us for a while.

Reply:
Also this method enabled color retrieval in light mode, but it's not doing well with dark mode. A workaround that would work from iOS 13 and should be OK for DPs to customise as well would be to define the colors as assets and initialize them by name. This would work fine for both UIColors and Colors, but it's a bit of a hassle to implement right now. @BartlomiejSopalaKarhoo found a solution that should work starting from iOS 14, so we'll postpone dark mode for a little longer.